tests/integration/psync2.tcl

```diff
@@ -123,6 +143,19 @@ start_server {} {
             }
         }
 
+        # For the newly detached master-replica chain (new master and existing replicas that were
+        # already connected to it, to get updated on the new replication id.
+        wait_for_condition 50 1000 {
```

Comment:
```suggestion
        # This is needed to avoid a race that can result in a full sync when a replica that already
        # got an updated repl id, tries to psync from one that's not yet aware of it.
        wait_for_condition 50 1000 {
```